### PR TITLE
Feature/fork project

### PR DESCRIPTION
**Pull request by PierreCrb**
fork project

Added the functionality to fork a project that is not linked to the currently logged in user ✅

### CODE REVIEW

**Comment by PierreCrb** [comment]
Sans titre

Add project name in url ! ✅

File: prisma/migrations/20201028102833/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url = "***"
```
Comment (shinework) [inline prisma/migrations/20201028102833/schema.prisma:6]: Please use an env var: 
```
url = env("DATABASE_URL")
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -12,14 +29,134 @@ import EditorErrorBoundary from '~components/errorBoundaries/EditorErrorBoundary
...
+      })
+    }
+  }
+
+  const saveProject = async () => {
+    if (session) {
+      if (props.id) {
+        let userCanEdit = false
+        const userProject = await checkUser(session.user.name)
+        userProject.project.map((e: Project) => {
```
Comment (shinework) [inline src/pages/index.tsx:119]: You can use the some function here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

const userCanEdit = userProject.project.some((e: Project) => (e.id === props.id))

**Review by shinework — CHANGES_REQUESTED** [review]

**Comment by PierreCrb** [comment]
demo

Login / Logout ✅
Clean code ✅
Fix slug url error ✅

**Review by shinework — APPROVED** [review]

**Pull request merged**